.github/workflows/gradle-check.yml

```diff
@@ -30,11 +30,14 @@ jobs:
       - name: Setup environment variables (PR)
         if: github.event_name == 'pull_request_target'
         run: |
+          echo "branch_name=$(jq --raw-output .pull_request.base.ref $GITHUB_EVENT_PATH)" >> $GITHUB_ENV
```

Comment:
I'd recommend we include the `event_name` as the way to differentiate between ad-hoc test executions associating with a pull request vs pushes that had to have a passing CI check prior to merging

Reply:
You mean to say to have an additional field `event` in the index mapping to easily filter?

Reply:
Done!